source/excerptexport/views.py

```diff
@@ -36,10 +37,13 @@ def new_excerpt_export(request):
 
 @login_required(login_url='/admin/')
 def create_excerpt_export(request):
-    viewContext = {}
```

Comment:
The current `viewContext` (the functionality of which I tried to preserve for now) is only useful for development/debugging anyway. The `viewContext` we finally want for `create_excerpt_export` will be creatable completely from the finished `ExtractionOrder`, so can be filled in at just one place in the code instead of scattered around like now.

In fact, this POST-request specific view shouldn't build a `viewContext` at all. Instead, it should redirect to a GET of a separate view of the newly created object. (See https://docs.djangoproject.com/en/1.7/intro/tutorial04/, first `polls/views.py` snipped and last bullet points of the 'rundown' following it)